services/color.md

```diff
@@ -0,0 +1,16 @@
+# Color
+
+Senses RGB colors
+
+    identifier: 0x1630d567
+    extends: _sensor
+
+## Registers
+
+    ro color @ reading {
+        red: u8
+        green: u8
+        blue: u8
+    }
+
+Detected color in the RGB color space.
```

Comment:
maybe do `u16 /` ?

Reply:
16 bits for each color channels?

Reply:
yes; there are these color sensors for calibrating photo machines, I suspect they are more than 8 bit per channel

Reply:
fixed.